src/components/page-components/SignUpPage/SignUpPage.tsx

```diff
@@ -12,12 +12,13 @@ import { sendToBackground } from "@plasmohq/messaging"
 import { useStorage } from "@plasmohq/storage/hook"
 
 import FormInput from "~components/Forms/PasswordInput/FormInput"
-import PasswordInput from "~components/Forms/PasswordInput/PasswordInput"
+import PasswordInput from "~components/Forms/PasswordInput/PasswordInput/PasswordInput"
+import useStore from "~store/store"
 // import useStore from "~store/store"
```

Comment:
looks like a new import was automatically added when you call `useStore` but an old import was left commented out